index.html

```diff
@@ -23,22 +23,147 @@ <h1>Hey There.i'm Said</h1>
+      <h1 class="title2">My Recent Works</h1>
+      <section class="work project0">
+        <div class="info-project">
+          <h2>Multi-Post Stories</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <ul class="tags">
+          <li class="css">css</li>
+          <li class="html">html</li>
+          <li class="boot">bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <form class="check">
+          <button type="button" class="click">See Project</button>
+        </form>
+      </section>
+      <section class="work project1">
+        <div class="linfo">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="lang">
+            <li class="ht">html</li>
+            <li class="bo">bootstrap</li>
+            <li class="ru">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="but">See Project</button>
+        </form>
+      </section>
+      <section class="work projectq">
+        <div class="linfoq">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langq">
+            <li class="htq">html</li>
+            <li class="boq">bootstrap</li>
+            <li class="ruq">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butq">See Project</button>
+        </form>
+      </section>
+      <section class="work projectw">
+        <div class="linfow">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langw">
+            <li class="htw">html</li>
+            <li class="bow">bootstrap</li>
+            <li class="ruw">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butw">See Project</button>
+        </form>
+      </section>
+      <section class="work projectX">
+        <div class="linfoe">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="leng">
+            <li class="hte">html</li>
+            <li class="strap">bootstrap</li>
+            <li class="rube">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="bute">See Project</button>
+        </form>
+      </section>
+      <section class="work about">
```

Comment:
- [ ] It is good that you have used <section> for the about-me section. However, it should be outside the `works` section on its own. For example, the skeleton would look like:
```
<body>
   <header class='header'></header>
   <section class='headline'></section>
   <section class='works'></section>
   <section class='about-me'></section>
   <footer class='footer'></footer>   
</body>
```

Reply:
@gtekle Thank you for reviewing my project, I will make the necessary corrections as suggested